### ISSUE TITLE
CentOS7 packages in opensuse repo out of date

### ISSUE THREAD

**Issue by lkeijser** [ClusterLabs/crmsh#132]
Hi,

Apologies if you're not the maintainer. Haven't done too much research, I admit :)

Anyway, the CentOS7 packages are out of date:

```
# yum update python
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: ftp.nluug.nl
 * extras: mirror.denit.net
 * updates: mirror.denit.net
Resolving Dependencies
--> Running transaction check
---> Package python.x86_64 0:2.7.5-18.el7_1.1 will be updated
---> Package python.x86_64 0:2.7.5-34.el7 will be an update
--> Processing Dependency: python-libs(x86-64) = 2.7.5-34.el7 for package: python-2.7.5-34.el7.x86_64
--> Running transaction check
---> Package python-libs.x86_64 0:2.7.5-18.el7_1.1 will be updated
---> Package python-libs.x86_64 0:2.7.5-34.el7 will be an update
--> Processing Dependency: /bin/python for package: python-parallax-1.0.1-10.1.noarch
--> Finished Dependency Resolution
Error: Package: python-parallax-1.0.1-10.1.noarch (@/python-parallax-1.0.1-10.1.noarch)
           Requires: /bin/python
           Removing: python-2.7.5-18.el7_1.1.x86_64 (@updates)
               Not found
           Updated By: python-2.7.5-34.el7.x86_64 (base)
               Not found
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest

[root@backend2 ~]# rpm -q python
python-2.7.5-18.el7_1.1.x86_64
```

I suspect it's relatively easy to fix though. Could you also make the srpm available?

Thanks!

**Comment by krig** [comment]
Hi!

I have released a new version of crmsh and I've updated the python-parallax packages so that they build on centOS again. See the OBS here:

https://build.opensuse.org/package/show/network:ha-clustering:Stable/crmsh

CentOS7 packages are available here:

http://download.opensuse.org/repositories/network:/ha-clustering:/Stable/CentOS_CentOS-7/

Hopefully this fixes any issues you've had. Though I have to confess that I don't quite understand what it is yum is complaining about. python-parallax shouldn't depend on a particular python version.
